tests/meson.build

```diff
@@ -140,6 +140,7 @@ tests = {
   'uri': {'suites': ['softokn', 'softhsm', 'kryoptic']},
   'ecxc': {'suites': ['softhsm', 'kryoptic']},
   'cms': {'suites': ['softokn', 'kryoptic']},
+  'pinlock': {'suites': ['kryoptic']},
```

Comment:
It does not work with others? At all? Or do they have just different amount of failed attempts?

Reply:
I did not have time to test :-)

Reply:
Ok tested now, but both softokn and softhsm fail the tests with an odd error:
error: PKCS11 function C_OpenSession failed: rv = CKR_SLOT_ID_INVALID (0x3)
and never reach final try of course.

Let me investigate if my setup is incorrect in those cases.

Reply:
Interesting. I am not sure if the softokens support the pin counter though as its more like security through obscurity given that the DB can be always copied or backed up.
If it would be too complicated, just a comment that it does not work would be probably good.